eservice/docs/sservice.md

```diff
@@ -31,20 +31,21 @@ The storage service is configured through a Toml configuration file generally pl
 
 The storage service supports several operations for interacting with the block store.
 
-| URL          | Method | Request Body     | Response Body            | Description                                    |
-|:-------------|:-------|:-----------------|:-------------------------|------------------------------------------------|
-| /block/<id>  | GET    | None             | application/octet-string | Return the requested block                     |
-| /block/list  | GET    | None             | application/json         | List of all known blocks                       |
-| /block/check | POST   | application/json | application/json         | Size and expiration of requested blocks        |
-| /block/store | POST   | multipart/form   | application/json         | Store multiple blocks, return proof of storage |
-| /info        | GET    | None             | application/json         | Request information about the service          |
-| /shutdown    | GET    | None             | None                     | Request to shutdown the service                |
+| URL             | Method | Request Body     | Response Body            | Description                                    |
+|:----------------|:-------|:-----------------|:-------------------------|------------------------------------------------|
+| /block/get/<id> | GET    | None             | application/octet-string | Return the requested block                     |
```

Comment:
From a REST perspective, the /get in the path seems a bit surprising? but i guess wsgi forced you to have a separate prefix to distinguish from list, store & check?